src/keyboard.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import CKEditorError from './ckeditorerror.js';
+
+/**
+ * Set of utils related to keyboard support.
+ *
+ * @namespace utils.keyboard
+ */
+
+/**
+ * Object with `keyName => keyCode` pairs.
+ *
+ * Contains:
+ *
+ * * `a-z`,
+ * * `0-9`,
+ * * `arrow(left|up|right|bottom)`,
+ * * `backspace`, `delete`, `enter`,
+ * * `ctrl`, `cmd`, `shift`, `alt`.
+ *
+ * @member {Object} utils.keyboard.knownKeyNames
+ */
+export const knownKeyNames = generateKnownKeyNames();
```

Comment:
Hm... It underlines that these are not all key names. But ok, I'll remove the word and improve the documentation.